<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* QWidget#centralwidget {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(164, 241, 202, 255), stop:0.142 rgba(205, 255, 255, 255), stop:0.285 rgba(176, 234, 42, 255), stop:0.428 rgba(222, 186, 255, 255), stop:0.571 rgba(244, 214, 96, 255), stop:0.714 rgba(8, 228, 255, 255), stop:0.857 rgba(255, 169, 112, 255));
} */

QWidget#centralwidget {
	background-color: #302554;
}

QGroupBox {
	color: white;
}

QMenuBar {
	background-color: #717171;
	color: #FFFFFF;
}
QMenuBar::item::selected {
	background-color: #FF0000;
	font: bold 12pt &quot;Arial&quot;;
}
QMenu {
	background-color: #717171;
	color: #FFFFFF;
	font: bold 12pt &quot;Arial&quot;;
    }
QMenu::item::selected {
	background-color: #2E2E2E;
	color: #FFFF00;
	border: 3px solid;	
	border-left-color:#FFFF00;		
	border-top-color: #00000000;
	border-bottom-color: #00000000;
	border-right-color: #00000000;
	font: bold 12pt &quot;Arial&quot;;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lb_background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>391</width>
      <height>221</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>background</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lb_element">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_weapon_dps">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>arma dps 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_dps_build">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>281</width>
      <height>30</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>t√≠tulo build dps</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_artifact_dps_1">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>270</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>artifact</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_artifact_dps_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>270</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>artifact</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_weapon_dps_or">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>arma dps 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_artifact_dps_1_or">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>360</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>artifact</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_artifact_dps_2_or">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>360</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>artifact</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>450</y>
      <width>304</width>
      <height>50</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_artifact_dps_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>510</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>artifact</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_artifact_dps_2_or_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>600</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>artifact</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_artifact_dps_1_or_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>600</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>artifact</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_weapon_dps_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>arma dps 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_artifact_dps_4">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>510</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>artifact</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_weapon_dps_or_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>600</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>arma dps 1</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>389</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAnemo">
    <property name="title">
     <string>Anemo</string>
    </property>
    <addaction name="actionJean"/>
    <addaction name="actionSucrose"/>
    <addaction name="actionTraveler_Anemo"/>
    <addaction name="actionVenti"/>
    <addaction name="actionXiao"/>
   </widget>
   <widget class="QMenu" name="menuDendro">
    <property name="title">
     <string>Dendro</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuElectro">
    <property name="title">
     <string>Electro</string>
    </property>
    <addaction name="actionBeidou_2"/>
    <addaction name="actionFischl_2"/>
    <addaction name="actionKeqing_2"/>
    <addaction name="actionLisa"/>
    <addaction name="actionRazor_2"/>
   </widget>
   <widget class="QMenu" name="menuGeo">
    <property name="title">
     <string>Geo</string>
    </property>
    <addaction name="actionAlbedo"/>
    <addaction name="actionningguang"/>
    <addaction name="actionNoelle"/>
    <addaction name="actionTraveler_Geo"/>
    <addaction name="actionZhongli"/>
   </widget>
   <widget class="QMenu" name="menuHydro">
    <property name="title">
     <string>Hydro</string>
    </property>
    <addaction name="actionBarbara"/>
    <addaction name="actionMona"/>
    <addaction name="actionTartaglia"/>
    <addaction name="actionXingqiu"/>
   </widget>
   <widget class="QMenu" name="menuPyro">
    <property name="title">
     <string>Pyro</string>
    </property>
    <addaction name="actionAmber"/>
    <addaction name="actionBennett"/>
    <addaction name="actionDiluc"/>
    <addaction name="actionHu_Tao"/>
    <addaction name="actionKlee"/>
    <addaction name="actionXingling"/>
    <addaction name="actionXinyan"/>
   </widget>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuCryo">
    <property name="title">
     <string>Cryo</string>
    </property>
    <addaction name="actionChongyun"/>
    <addaction name="actionDiona"/>
    <addaction name="actionGanyu"/>
    <addaction name="actionKaeya"/>
    <addaction name="actionQiqi"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuAnemo"/>
   <addaction name="menuCryo"/>
   <addaction name="menuDendro"/>
   <addaction name="menuElectro"/>
   <addaction name="menuGeo"/>
   <addaction name="menuHydro"/>
   <addaction name="menuPyro"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string> Exit</string>
   </property>
  </action>
  <action name="actionJean">
   <property name="text">
    <string> Jean</string>
   </property>
  </action>
  <action name="actionSucrose">
   <property name="text">
    <string> Sucrose</string>
   </property>
  </action>
  <action name="actionTraveler_Anemo">
   <property name="text">
    <string> Traveler (Anemo)</string>
   </property>
  </action>
  <action name="actionVenti">
   <property name="text">
    <string> Venti</string>
   </property>
  </action>
  <action name="actionXiao">
   <property name="text">
    <string> Xiao</string>
   </property>
  </action>
  <action name="actionBeidou">
   <property name="text">
    <string>Beidou</string>
   </property>
  </action>
  <action name="actionFischl">
   <property name="text">
    <string>Fischl</string>
   </property>
  </action>
  <action name="actionKeqing">
   <property name="text">
    <string>Keqing</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Lisa</string>
   </property>
  </action>
  <action name="actionRazor">
   <property name="text">
    <string>Razor</string>
   </property>
  </action>
  <action name="actionBeidou_2">
   <property name="text">
    <string> Beidou</string>
   </property>
  </action>
  <action name="actionFischl_2">
   <property name="text">
    <string> Fischl</string>
   </property>
  </action>
  <action name="actionKeqing_2">
   <property name="text">
    <string> Keqing</string>
   </property>
  </action>
  <action name="actionLisa">
   <property name="text">
    <string> Lisa</string>
   </property>
  </action>
  <action name="actionRazor_2">
   <property name="text">
    <string> Razor</string>
   </property>
  </action>
  <action name="actionAlbedo">
   <property name="text">
    <string> Albedo</string>
   </property>
  </action>
  <action name="actionningguang">
   <property name="text">
    <string> Ningguang</string>
   </property>
  </action>
  <action name="actionNoelle">
   <property name="text">
    <string> Noelle</string>
   </property>
  </action>
  <action name="actionTraveler_Geo">
   <property name="text">
    <string> Traveler (Geo)</string>
   </property>
  </action>
  <action name="actionZhongli">
   <property name="text">
    <string> Zhongli</string>
   </property>
  </action>
  <action name="actionBarbara">
   <property name="text">
    <string> Barbara</string>
   </property>
  </action>
  <action name="actionMona">
   <property name="text">
    <string> Mona</string>
   </property>
  </action>
  <action name="actionTartaglia">
   <property name="text">
    <string> Tartaglia</string>
   </property>
  </action>
  <action name="actionXingqiu">
   <property name="text">
    <string> Xingqiu</string>
   </property>
  </action>
  <action name="actionAmber">
   <property name="text">
    <string> Amber</string>
   </property>
  </action>
  <action name="actionBennett">
   <property name="text">
    <string> Bennett</string>
   </property>
  </action>
  <action name="actionDiluc">
   <property name="text">
    <string> Diluc</string>
   </property>
  </action>
  <action name="actionHu_Tao">
   <property name="text">
    <string> Hu Tao</string>
   </property>
  </action>
  <action name="actionKlee">
   <property name="text">
    <string> Klee</string>
   </property>
  </action>
  <action name="actionXingling">
   <property name="text">
    <string> Xingling</string>
   </property>
  </action>
  <action name="actionXinyan">
   <property name="text">
    <string> Xinyan</string>
   </property>
  </action>
  <action name="actionChongyun">
   <property name="text">
    <string> Chongyun</string>
   </property>
  </action>
  <action name="actionDiona">
   <property name="text">
    <string> Diona</string>
   </property>
  </action>
  <action name="actionGanyu">
   <property name="text">
    <string> Ganyu</string>
   </property>
  </action>
  <action name="actionKaeya">
   <property name="text">
    <string> Kaeya</string>
   </property>
  </action>
  <action name="actionQiqi">
   <property name="text">
    <string> Qiqi</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
